CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- Update the contributing guide with fork instructions ([#8268](https://github.com/pyg-team/pytorch_geometric/pull/8268))
```

Comment:
I think for this change, we don't need to add an entry to the changelog as it doesn't impact PyG users
```suggestion
```